flytekit/clis/sdk_in_container/run.py

```diff
@@ -449,10 +449,14 @@ def to_click_option(
     # If a query has been specified, the input is never strictly required at this layer
     required = False if default_val and isinstance(default_val, ArtifactQuery) else required
 
+    if literal_converter.is_bool():
+        click_cli_paramater_name = f"--{input_name}/--no_{input_name}"
```

Comment:
are we able to support both? if the input is `my_input: int`, pyflyte should accept both `--no_my_input` and `--no-my-input`.
```suggestion
        click_cli_parameter_name = [f"--{input_name}/--no_{input_name}", f"--{input_name}/--no-{input_name.replace('_', '-')}"]
```

Reply:
It looks like we can :+1: